src/components/Api.js

```diff
@@ -0,0 +1,129 @@
+export default class Api {
+    constructor(options) {
+        // constructor body
+        this._baseURL = options.baseURL;
+        this._headers = options.headers;
+    }
+
+    getCards() {
+        return fetch(`${this._baseURL}/cards`, {
+            headers: this._headers
+        })
+        .then(res => {
```

Comment:
The code for processing the response is duplicated in all of the api methods. You should create a separate method `_processResponse` and use it instead. 

Other api methods will look like the following:

```js
return fetch(...).then(this._processResponse)
```

Reply:
Done! :) Thank you.